state/state.go

```diff
@@ -1760,15 +1763,7 @@ func (st *State) addMachineWithPlacement(unit *Unit, placement *instance.Placeme
 		}
 		return st.AddMachineInsideNewMachine(template, template, data.containerType)
 	case directivePlacement:
-		// If a placement directive is to be used, do that here.
-		template := MachineTemplate{
-			Series:      unit.Series(),
-			Jobs:        []MachineJob{JobHostUnits},
-			Dirty:       true,
-			Constraints: *unitCons,
-			Placement:   data.directive,
-		}
-		return st.AddOneMachine(template)
+		return nil, errors.NotSupportedf("directly adding a machine for %s with a placement directive", unit.Name())
```

Comment:
This wording is confusing - in the docs any value for `--to` is called a placement directive, so this suggests that we couldn't use `--to lxd:0` here. Is that right?
Is this error a user or a programming error? I think it's the latter - can you make it a bit more specific in that case?

Reply:
reworded